frontend/src/utils/routes.ts

```diff
@@ -6,6 +6,7 @@ const ROUTES = {
   GROUP_CREATE: '/group/new',
   GROUP_EDIT: '/group/:groupId',
   SHIPMENT_LIST: '/shipments',
+  SHIPMENT_EDIT: '/shipment/:shipmentId',
```

Comment:
Suggestion (blocking): this URL path looks like a path for viewing the details of a shipment rather than editing one. Recommending changing to `/shipment/:id/edit`